### ISSUE TITLE
Dev::Self host smart chart not loading, be hide nginx proxy 

### ISSUE THREAD

**Issue by nextnaro** [binary-com/binary-bot#1070]
it work fine if direct access on localhos:8080

        proxy_pass http://127.0.0.1:8080;
        proxy_http_version 1.1;
        proxy_set_header Upgrade $http_upgrade;
        proxy_set_header Connection 'upgrade';
        proxy_set_header Host $host;
        proxy_cache_bypass $http_upgrade;

![image](https://user-images.githubusercontent.com/31925501/50634549-fd881180-0f89-11e9-9f57-301dc4dfa2cf.png)

**Comment by aaron-binary** [comment]
Hey nextnaro,

Could you explain your issue in more detail?

Regards,
Aaron

**Comment by nextnaro** [comment]
Hi @aaron-binary  i install latest version on my host.
i use nginx web server for reverse proxy  everything look working fined but only smartchart unable to show as captured.

why is use nginx reverse proxy?
because binary bot only can access on localhost it self(i dont know where to edit code or configuration).
my binary bot self host able to access on localhost:8080 and 127.0.0.1:8080 but, i cant brows to host ip address ,host name or domain name from another client. when browse showing "This site can’t be reached" http://192.168.110.128:8080 IP pingable.

**Comment by nextnaro** [comment]
yeh i found 

SmartCharts is both the name of the app ([charts.binary.com](https://charts.binary.com/)) and the charting library. You can install the library to your project via:

    yarn add @binary-com/smartcharts      # Release
    yarn add @binary-com/smartcharts@beta # Beta

**Important Note:** the license for the library is tied to the `binary.com` domain name; it will not work in github pages.

and *\tDomain lock: [\"127.0.0.1\",\"localhost\",\"binary.com\",\"binary.sx\",\"binary.me\"]\n.

**### how to fix this**
why is use nginx reverse proxy?
because binary bot only can access on localhost it self(i dont know where to edit code or configuration).
my binary bot self host able to access on localhost:8080 and 127.0.0.1:8080 but, i cant brows to host ip address ,host name or domain name from another client. when browse showing "This site can’t be reached" http://192.168.110.128:8080 IP pingable.

**Comment by nextnaro** [comment]
yes, i followed. but it working only localhost (brows on it self).
how do i access from client >> server?
i install working if local access. i cant brows from client.(by IP , domain same) change to port 80 not work also. give up. 
sorry on kept repeating question and my English.
